tools/integration_tests/mounting/gcsfuse_test.go

```diff
@@ -551,3 +558,54 @@ func (t *GcsfuseTest) HelpFlags() {
 		ExpectEq(nil, err, "case %d\nOutput:\n%s", i, output)
 	}
 }
+
+func (t *GcsfuseTest) RelativeLogFilePath() {
+
+	var err error
+	var homeDir string
+
+	// create a temp file in current working directory
+	currentDirTestFile := filepath.Join(t.dir, "test.txt")
+	_, err = os.Create(currentDirTestFile)
+	AssertEq(nil, err)
+	defer os.Remove(currentDirTestFile)
+
+	//create a temp log file in home directory
+	homeDir, err = os.UserHomeDir()
+	AssertEq(nil, err)
+
+	homeTestFile := filepath.Join(homeDir, "test_home.json")
+	_, err = os.Create(homeTestFile)
+	defer os.Remove(homeTestFile)
+
+	// Specify log file in different way.
+	testCases := []struct {
+		extraArgs []string
+		env       []string
+	}{
+		// without --foreground flag, relative path
+		0: {
+			extraArgs: []string{"--log-file", "test.txt"},
+		},
+
+		// without --foreground flag, relative with ./
+		1: {
+			extraArgs: []string{"--log-file", "./test.txt"},
+		},
+
+		// without --foreground flag, path with tilda
+		2: {
+			extraArgs: []string{"--log-file", "~/test_home.json"},
+			env:       []string{fmt.Sprintf("HOME=%s", homeDir)},
+		},
+	}
+
+	for _, tc := range testCases {
```

Comment:
all the tests are for log-file only but you have changed key-file also.
Pls add tests for a single key-file and also if both key-file and log-file are provided.

Reply:
done.